product_development.md

```diff
@@ -0,0 +1,96 @@
+# Product development
+
+## Documents & Artifacts
+
+### Kanban board
+
+The Kanban board is the main tool of communication between stakeholders and the different product development units.
+
+The Kanban board is owned by the product management team.
+
+Engineers are responsible for maintaining cards on the Kanban board once they have pulled them into the backlog. Stories are always assigned when they are in progress, but never while they are in the backlog.
+
+All work that is done in either product, design or engineering should be reflected on the board. Stakeholders may have tickets on the board too.
+
+### New feature tickets
+
+The requirements engineering and conception of new features is owned by the product and design team.
+
+### Growth opportunity
+
+Anything that can help a department (not only product related) to improve their performance and the performance of our business.
+
+Growth opportunities should align with our long term product vision.
+
+Depending on the complexity those opportunities might be developed in small increments prioritizing value delivery.
+
+### OKR initiatives
+
+Strategic initiatives developed with a department and prioritized in close coordination with an objective owner. Those initiatives could be features and also research tasks, surveys, content work and more.
+
+They are prioritized over anything else to ensure the success of our OKRs.
+
+OKR initiatives should be in line with our long-term product vision.
+
+### Operational hiccups
+
+Operational hiccups are issues that disturb a stakeholders operation or hinder their ability to exercise their responsibilities. They usually require some kind of technical support or facilitation and should have some level of urgency similar to bugs but are outside of the engineering responsibilities.
```

Comment:
I'm not sure what this is and who is resolving it.

Reply:
I am a bit lost in here...

1. How would the urgency be determined?  
2. Who is the responsible to resolve it? 

In many cases, each stakeholder have a different understanding and expectation of that is hindering their ability to exercise their responsibilities.

Reply:
Good point, I will elaborate on that a bit more